committeeoversightapp/management/commands/import_data.py

```diff
@@ -12,9 +12,6 @@
 from opencivicdata.legislative.models import Event, EventSource, EventParticipant
 from committeeoversightapp.models import HearingCategory, Committee
 
-jurisdiction_id = 'ocd-jurisdiction/country:us/legislature'
-bad_rows = []
-
 def get_committee(committee_name, parent):
```

Comment:
I'd like to see this as a class function, too - as you did with `new_committee`.